package.json

```diff
@@ -24,6 +24,7 @@
     "@babel/polyfill": "^7.8.7",
     "axios": "^0.19.2",
     "body-parser": "^1.19.0",
+    "connect-mongodb-session": "^2.3.1",
```

Comment:
Not sure why this is needed considering I don't see it utilized.

Reply:
good find. I think I had it from a previous session?